docs/docs/adrs/intro.md

```diff
@@ -40,4 +40,5 @@ To suggest an ADR, please make use of the [ADR template](./adr-template.md) prov
 | [007](./adr-007-pause-unbonding-on-eqv-prop.md) | Pause validator unbonding during equivocation proposal | Proposed |
 | [008](./adr-008-throttle-retries.md) | Throttle with retries | Accepted, In-progress |
 | [009](./adr-009-soft-opt-out.md) | Soft Opt-out | Accepted, Implemented |
-| [009](./adr-010-standalone-changeover.md) | Standalone to Consumer Changeover | Accepted, Implemented |
+| [010](./adr-010-standalone-changeover.md) | Standalone to Consumer Changeover | Accepted, Implemented |
+| [013](./adr-013-equivocation-slashing.md) | Slashing on the provider for consumer equivocation | Proposed |
```

Comment:
Using `013` number as I see `012` is already [on its way](https://github.com/cosmos/interchain-security/pull/1229#discussion_r1309888523).